src/App.js

```diff
@@ -609,45 +644,70 @@ class App extends Component {
     this.updateBufferTimer = setTimeout(() => {
       const buffer = this.markupBuffer;
       this.markupBuffer = [];
-      this.updateBufferSingle(null, buffer);
+      this.processBuffer(null, buffer);
     }, this.bufferIntervalMillis());
   }
 
   bufferIntervalMillis() {
     return 16; // 60 fps
   }
 
-  updateBufferSingle(actualText, buffer) {
+  /*
+   * Given the buffer of inputs, determines the new state, collects the side effects,
+   * then applies the side effects and updates the state.
+   *
+   * @actualText param is not used - probably should be removed
```

Comment:
It looks to me like `actualText` is passed to `getNewStateAndSideEffectsForBuffer` and then passed to `matchSplitText` and then used by `const actualTextChars`, which is used in a few places. Have I misunderstood this comment?

Reply:
@didoesdigital if my logic is correct, all use cases of processBuffer result in the `actualText` parameter being ignored and never used as given.

the getNewStateAndSideEffectsForBuffer function will overwrite the actualText value any time when the second parameter `buffer` evaluates to `true` in an if statement.

Maybe it's ok to remove it in the next round, as the parameter is not doing anything.

Reply:
Oh I see what you mean. Yep, we can clean that up after.